cmds/core/ping/ping_other.go

```diff
@@ -0,0 +1,12 @@
+// +build !linux
```

Comment:
I don't like this, but at least we have ping6 on Linux, and the build for other OSes doesn't fail